server/core/src/main/scala/org.apache.daffodil.debugger.dap/ErrorEvents.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.debugger.dap
+
+import com.microsoft.java.debug.core.protocol.Events.DebugEvent
+
+/** Case classes for errors that we want to relay back to the extension */
+object ErrorEvents {
+  case class UnexpectedError(message: String) extends DebugEvent("daffodil.error.unexpected")
+  case class UnhandledRequest(message: String) extends DebugEvent("daffodil.error.requestunhandled")
+  case class LaunchArgsParseError(message: String) extends DebugEvent("daffodil.error.launchargparse")
```

Comment:
DebugEvent really doesn't need this string arg. You can just do 
```
lazy val event = Misc.getNameFromClass(this)
```
since you just need to display these strings as part of the abort info on the client side. The class name will do just fine.

Reply:
I will remove the message param from all ErrorEvents, that makes a look of sense. I will make case objects instead of case classes as well.

Reply:
@mbeckerle so would something like `vscode.window.showErrorMessage(`debugger ${e.event}`)` be enough? It would display something like `debugger daffodil.error.request` in the bottom right. then the error information is inside of the terminal.

Reply:
@mbeckerle The above comment is what I latest commit does, let me if you would prefer something different.